server/protocols/jmap/src/main/java/org/apache/james/jmap/send/PostDequeueDecorator.java

```diff
@@ -81,7 +81,7 @@ public void done(boolean success) throws MailQueueException {
                     moveFromOutboxToSent(messageId, mailboxSession);
                     getMail().setAttribute(IS_DELIVERED, IS_DELIVERED);
                 } catch (MailShouldBeInOutboxException e) {
-                    LOG.info("Message does not exist on Outbox anymore, it could have already been sent {}", e);
+                    LOG.info("Message does not exist on Outbox anymore, it could have already been sent {}", e.toString());
```

Comment:
Why do you add "toString()"

Reply:
This is to work around a regression introduced in SLF4J 1.7.15. James Project is using SLF4J 1.7.25, which is affected.

See http://mailman.qos.ch/pipermail/slf4j-user/2017-August/001639.html and the follow-up message http://mailman.qos.ch/pipermail/slf4j-user/2017-August/001640.html

Reply:
I guess here we want to have the error with the full stacktrace so I would go for:
```LOG.info("Message does not exist on Outbox anymore, it could have already been sent", e);""

Reply:
:+1:

Reply:
Done